apps/desktop/ui/style.css

```
@@ -0,0 +1,4 @@
+html { font-family: system-ui, sans-serif; }
+body { margin: 0; }
+main { max-width: 48rem; margin: 0 auto; padding: 2rem; }
+button:focus-visible { outline: 2px solid currentColor; outline-offset: 2px; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)